integration/test/dashboard_json_test.go

```diff
@@ -211,20 +208,50 @@ func (suite *DashboardJsonTestSuite) TestJsonExpression() {
 	}
 
 	if isRestFailed {
-		assert.Fail(suite.T(), fmt.Sprintf("Rest Test validation is failed. Pls check logs above. Count of Missing Rest counters %d", len(restErrorInfoList)))
+		t.Errorf("Rest Test validation is failed. Pls check logs above. Count of Missing Rest counters %d", len(restErrorInfoList))
 	} else {
 		log.Info().Msg("Rest Validation looks good!!")
 	}
 
 	// Fail if either Rest or Zapi collectors have failures
 	if isZapiFailed {
-		assert.Fail(suite.T(), fmt.Sprintf("Zapi Test validation is failed. Pls check logs above. Count of Missing Zapi counters %d", len(zapiErrorInfoList)))
+		t.Errorf("Zapi Test validation is failed. Pls check logs above. Count of Missing Zapi counters %d", len(zapiErrorInfoList))
 	} else {
 		log.Info().Msg("Zapi Validation looks good!!")
 	}
 }
 
-func ShouldSkipDashboard(path string) bool {
+func waitForCollectors(t *testing.T) {
+	log.Info().Msg("Exclude map info")
+	log.Info().Str("Exclude Mapping", fmt.Sprint(counterMap)).Msg("List of counter")
+	log.Info().Msg("Wait for data to be collected")
+	countersToCheck := []string{
+		"copy_manager_kb_copied",
+		"qos_read_latency",
+		"qos_volume_read_data",
+		"qos_volume_read_data",
```

Comment:
why 2 times? and why only few counters are 2 times?

Reply:
👍 1 too separate lists got merged and I missed  the dups